roles/openshift_node_problem_detector/tasks/uninstall.yaml

```diff
@@ -1,35 +1,35 @@
 ---
 - name: Ensure the node problem detector is absent
   oc_obj:
-    namespace: "{{ openshift_node_problem_detector_namespace }}"
+    namespace: openshift-node-problem-detector
```

Comment:
I think in uninstall playbook, you ca just delete the namespace and that will delete everything else inside it, so can remove everything else, that means less code in here.